pkg/servicecentral/it/client_manual_test.go

```diff
@@ -0,0 +1,69 @@
+package it
+
+import (
+	"encoding/json"
+	"net/url"
+	"os"
+	"testing"
+
+	"github.com/atlassian/voyager/pkg/servicecentral"
+	"github.com/atlassian/voyager/pkg/util"
+	"github.com/atlassian/voyager/pkg/util/auth"
+	"github.com/atlassian/voyager/pkg/util/logz"
+	"github.com/atlassian/voyager/pkg/util/pkiutil"
+	"github.com/atlassian/voyager/pkg/util/testutil"
+	"github.com/stretchr/testify/require"
+	"k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+const (
+	defaultServiceCentralURL = "https://services.prod.atl-paas.net"
+	fakeTestUsername         = "fcobb"
+)
+
+// You can run this file from root using
+// `go test -v pkg/servicecentral/it/client_manual_test.go`
+
+func TestGetServiceAttributes(t *testing.T) {
+	t.Parallel()
+
+	// Prepare ASAP secrets from Kubernetes Secret
+	asapCreatorSecret := getSecret(t)
+	ctx := testutil.ContextWithLogger(t)
+	testLogger := logz.RetrieveLoggerFromContext(ctx)
+	testUser, authErr := auth.Named(fakeTestUsername)
+	require.NoError(t, authErr)
+	asapConfig, asapErr := pkiutil.NewASAPClientConfigFromKubernetesSecret(asapCreatorSecret)
+	require.NoError(t, asapErr)
+
+	client := util.HTTPClient()
+	c := servicecentral.NewServiceCentralClient(testLogger, client, asapConfig, parseURL(t, defaultServiceCentralURL))
+
+	// Get Service Attributes
+	resp, err := c.GetServiceAttributes(ctx, auth.ToOptionalUser(testUser), "slime")
+	require.NoError(t, err)
+
+	testLogger.Sugar().Infof("Number of returned attributes: %v", len(resp))
+	testLogger.Sugar().Infof("Attributes: %#v", resp)
```

Comment:
Use `t.Logf()` directly. And perhaps can actually just remove this logging? Can be added for debugging when needed.

Reply:
There should be some assertions on `resp`. Just printing the result is not very useful for automated testing.

Reply:
My understanding was that this is a manual test of the logic rather than an integration test. Does your comment still hold true in the case where its not an integration test?

Reply:
What exactly is manually tested? Why cannot those manual checks be encoded here? If you decide to keep logging, use `t` for it directly in tests, not zap.

Reply:
Heres where I took this idea from: https://github.com/atlassian/voyager/blob/master/pkg/pagerduty/it/client_manual_test.go#L28